### PR TITLE
Unit Tests for EncryptMetaDataReviseEntry

### LINKED ISSUE #28531: Add more unit tests for EncryptMetaDataReviseEntry
Hi community,

This issue is for https://github.com/apache/shardingsphere/discussions/28487.

## Aim
Add more unit tests for EncryptMetaDataReviseEntry to test its public functions to improve unit test coverage.

Unit test coverage can be viewed at: https://app.codecov.io/gh/apache/shardingsphere

## Basic Qualifications
- Java
- Maven
- Junit

### PR DESCRIPTION
**Pull request by shubham-j-sde**
Fixes #28531 .

Changes proposed in this pull request:
  -

---

Before committing this PR, I'm sure that I have checked the following options:
- [x] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [x] I have self-reviewed the commit code.
- [x] I have (or in comment I request) added corresponding labels for the pull request.
- [x] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [ ] I have made corresponding changes to the documentation.
- [x] I have added corresponding unit tests for my changes.

### CODE REVIEW

**Comment by yx9o** [comment]
Hi @shubham-j-sde , thank you for your contribution, there are some formats that can be optimized, you can refer to other Tests.

**Comment by shubham-j-sde** [comment]
Thanks, I'll check and commit again

File: features/encrypt/core/src/test/java/org/apache/shardingsphere/encrypt/metadata/reviser/EncryptMetaDataReviseEntryTest.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+import org.apache.shardingsphere.encrypt.api.config.EncryptRuleConfiguration;
+import org.apache.shardingsphere.encrypt.api.config.rule.EncryptTableRuleConfiguration;
+import org.apache.shardingsphere.encrypt.metadata.reviser.column.EncryptColumnExistedReviser;
+import org.apache.shardingsphere.encrypt.metadata.reviser.column.EncryptColumnNameReviser;
+import org.apache.shardingsphere.encrypt.metadata.reviser.index.EncryptIndexReviser;
+import org.apache.shardingsphere.encrypt.rule.EncryptRule;
+import org.junit.jupiter.api.Test;
+
+public class EncryptMetaDataReviseEntryTest {
```
Comment (yx9o) [inline features/encrypt/core/src/test/java/org/apache/shardingsphere/encrypt/metadata/reviser/EncryptMetaDataReviseEntryTest.java:36]: `public` is not required.

File: features/encrypt/core/src/test/java/org/apache/shardingsphere/encrypt/metadata/reviser/EncryptMetaDataReviseEntryTest.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        assertTrue(indexReviser.isPresent());
+        assertThat(indexReviser.get().getClass(), is(EncryptIndexReviser.class));
+    }
+    
+    @Test
+    void assertGetColumnNameReviser() {
+        Optional<EncryptColumnNameReviser> columnNameReviser = reviseEntry.getColumnNameReviser(rule, "test_table");
+        assertTrue(columnNameReviser.isPresent());
+        assertThat(columnNameReviser.get().getClass(), is(EncryptColumnNameReviser.class));
+
```
Comment (yx9o) [inline features/encrypt/core/src/test/java/org/apache/shardingsphere/encrypt/metadata/reviser/EncryptMetaDataReviseEntryTest.java:61]: Remove unnecessary blank lines.

**Review by tuichenchuxin — APPROVED** [review]

**Pull request merged**